teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(name, age, _parent_permission, specialization)
+    super(name, age, parent_permission: true)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same here, In initialize method you but the default values for the inherited variables the wrong way. So please fix that.